### PR TITLE
Added missing food emojis

### PR DESCRIPTION
**Pull request by diggabyte**
Adds a few newer food emojis to the data set.

### CODE REVIEW

File: LoopKitUI/CarbKit/FoodEmojiDataSource.swift [hunk 1]
```diff
@@ -97,17 +119,21 @@ private class FoodEmojiDataSource: EmojiDataSource {
         var slow = [
             "🍕", // pizza
             "🥑", // avocado
+            "🥚", // egg
```
Comment (diggabyte) [inline LoopKitUI/CarbKit/FoodEmojiDataSource.swift:122]: Source: ChatGPT

> Eggs are considered a slow-release protein source. Slow-release proteins are those that are digested and absorbed slowly by the body, providing a sustained release of amino acids over several hours. Eggs contain high-quality protein, which means they provide all of the essential amino acids needed by the body.
The protein in eggs is primarily found in the egg white, which is made up of approximately 90% water and 10% protein. The protein in egg whites is slow-release, meaning that it is digested and absorbed slowly by the body, providing a sustained release of amino acids over several hours.

File: LoopKitUI/CarbKit/FoodEmojiDataSource.swift [hunk 2]
```diff
@@ -50,13 +55,18 @@ private class FoodEmojiDataSource: EmojiDataSource {
             "🍟", // french fries
             "🍳", // cooking
             "🍲", // pot of food
+            "🥘", // pan of food
             "🍱", // bento box
             "🍛", // curry rice
             "🍜", // steaming bowl (ramen / noodle soup)
             "🍠", // roasted sweet potato
             "🍤", // fried shrimp
+            "🦪", // oyster
```
Comment (diggabyte) [inline LoopKitUI/CarbKit/FoodEmojiDataSource.swift:64]: Source: ChatGPT

> Oysters are a good source of protein, but they are not typically considered a slow-release protein source.

File: LoopKitUI/CarbKit/FoodEmojiDataSource.swift [hunk 3]
```diff
@@ -97,17 +119,21 @@ private class FoodEmojiDataSource: EmojiDataSource {
...
             "🍖", // meat on bone
             "🍗", // poultry leg
             "🥓", // bacon
             "🍔", // hamburger
             "🌭", // hot dog
             "🌯", // burrito
             "🍝", // spaghetti
-            "🥩" // cut of meat
+            "🥩", // cut of meat
+            "🧈", // butter
```
Comment (diggabyte) [inline LoopKitUI/CarbKit/FoodEmojiDataSource.swift:135]: Butter is high-fat content, slow release

File: LoopKitUI/CarbKit/FoodEmojiDataSource.swift [hunk 4]
```diff
@@ -97,17 +119,21 @@ private class FoodEmojiDataSource: EmojiDataSource {
         var slow = [
             "🍕", // pizza
             "🥑", // avocado
+            "🥚", // egg
             "🥜", // peanuts
             "🌰", // chestnut
             "🧀", // cheese wedge
+            "🫕", // fondue
```
Comment (diggabyte) [inline LoopKitUI/CarbKit/FoodEmojiDataSource.swift:126]: Fondues is high-fat content, slow release

File: LoopKitUI/CarbKit/FoodEmojiDataSource.swift (hunk 1 above)
Comment (diggabyte) [reply LoopKitUI/CarbKit/FoodEmojiDataSource.swift:122]: If anyone has any thoughts on whether egg should be in medium or slow, this is the one I was most unsure about..

**Comment by ps2** [comment]
Thanks. If anyone has any concerns about the categories for the newly added emojis, please submit a new PR showing what you think should change.

**Comment by diggabyte** [comment]
Shout out to @alexanng for the assistance on this one! 🙏

**Pull request merged**